public/js/controllers/charts_controller.js

```diff
@@ -25,30 +29,49 @@ function intComma (amount) {
   return amount.toLocaleString(undefined, { maximumFractionDigits: 0 })
 }
 
+function blockReward (height) {
+  if (height >= stakeValHeight) return baseSubsidy * Math.pow(100 / 101.0, Math.floor(height / subsidyInterval))
```

Comment:
I'd feel better if there was a comment referring to MulSubsidy and DivSubsidy.  Being that same on all networks presently is convenient, but it's certainly possible that will change.

Reply:
I'll just use the actual net params then.